pkg/server/app/app.go

```diff
@@ -19,6 +19,9 @@ type Operations interface {
 	Create(user *storage.User, app *App) error
 	Logs(user *storage.User, appName string, lines int64, follow bool) (io.ReadCloser, error)
 	Info(user *storage.User, appName string) (*Info, error)
+	TeamName(appName string) (string, error)
+	Meta(appName string) (*App, error)
+	HasPermission(user *storage.User, appName string) bool
```

Comment:
What do you think about `Get` ? 
I'm not sure because that method returns only information about the app, not a powerful struct with useful methods. BUT, is better than Meta? (or not? 🤔 )